tests/infer/test_inference.py

```diff
@@ -213,6 +225,7 @@ def guide():
     Trace_ELBO,
     TraceGraph_ELBO,
     TraceEnum_ELBO,
+    xfail_param(RenyiELBO, reason="Assertation errors at alpha_q, beta_q", run=False),
```

Comment:
hm this is worrisome, we can leave this as is until i have time to dive deeper into this unless @neerajprad wants to take a look

Reply:
@jpchen - I don't see anything obviously wrong atm, but I do want to dig deeper.